proj/src/main/webapp/test/chat.test.js

```diff
@@ -14,6 +14,77 @@ require("../app.js");
 let testHelper = require('./test-helper-functions.js');
 let chat = require('../chat.js');
 
+describe('creating and joining 1-on-1 chats', function() {
+    this.timeout(10000);
+    let testUsers = [{
+        email: 'test@test.com', 
+        password: 'test123!'
+    }, 
+    {
+        email: 'test2@test.com', 
+        password: 'test123!'
+    }];
+    let chatId;
+    const oneOnOneRef = firebase.database().ref('/chat/chats-1on1');
+
+    before(async function() {
+        for (let i = 0; i < testUsers.length; i++) {
+            await testHelper.createUserIfNotExisting(testUsers[i]);
+            await firebase.auth().signInWithEmailAndPassword(testUsers[i].email, testUsers[i].password);
+            testUsers[i]["uid"] = firebase.auth().currentUser.uid;
+            if (i == 1) {
+                chatId = oneOnOneRef.push().key;
+                await chat.createFriendChat(chatId, testUsers[0]["uid"]);
```

Comment:
Can you move this part into an `it("...",function() {...})" test case and do some checks that the database properly created the 1:1 chat properly?